config/helm/appmesh-controller/README.md

```diff
@@ -89,18 +89,49 @@ helm upgrade -i appmesh-controller eks/appmesh-controller \
 
 The [configuration](#configuration) section lists the parameters that can be configured during installation.
 
-**Note:** When using IRSA, make sure the Envoy proxies have the following IAM policies attached for Envoy to authenticate with AWS App Mesh and fetch it's configuration
+**Note**
+Make sure that the Envoy proxies have the following IAM policies attached for the Envoy to authenticate with AWS App Mesh and fetch it's configuration
 - https://raw.githubusercontent.com/aws/aws-app-mesh-controller-for-k8s/master/config/iam/envoy-iam-policy.json
 
-#### Setup IAM permissions manually on worker nodes
-If not setting up IAM role for service account, apply the IAM policies to your worker nodes:
+There are **2 ways** you can attach the above policy to the Envoy Pod  
+#### With IRSA     
+Download the Envoy IAM polocy  
+```
+curl -o envoy-iam-policy.json https://raw.githubusercontent.com/aws/aws-app-mesh-controller-for-k8s/master/config/iam/envoy-iam-policy.json
+```
+
+Create an IAM policy called AWSAppMeshEnvoyIAMPolicy
+```
+aws iam create-policy \
+    --policy-name AWSAppMeshEnvoyIAMPolicy \
+    --policy-document file://envoy-iam-policy.json
+```
+
+Take note of the policy ARN that is returned
+
+Create an IAM role for service account for the App Mesh Envoy, use the ARN from the step above. Ensure that Application Namespace already exists  
```

Comment:
We can also add that if the customer's Mesh enabled applications are already using IRSA then we can add the above envoy specific policies to the same policy.

Reply:
Will add a note specifying that Trust Relationship should be edited so that IAM role for the IRSA can be shared across namespaces